app/controllers/UsersController.scala

```diff
@@ -195,20 +202,32 @@ class UsersController @Inject()(messagesApi: MessagesApi,
           } { user =>
             val admin = user.admin
```

Comment:
I guess there's no need to create these local variables "admin", "password" and "superUser". "user.admin", "user.password" and "user.superUser" can be used directly where ever required.